script/child/DeployQWAaveV2.s.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity 0.8.23;
+
+import {DeployBase} from '../helpers/DeployBase.sol';
+
+import {QWManager} from 'contracts/QWManager.sol';
+import {QWRegistry} from 'contracts/QWRegistry.sol';
+import {QWAaveV2} from 'contracts/child/QWAaveV2.sol';
+import {Script} from 'forge-std/Script.sol';
+
+contract DeployQWAaveV2 is Script, DeployBase {
+  struct ConfigParams {
+    address aaveLendingPool;
+  }
+
+  QWAaveV2 public qwAaveV2;
+  QWRegistry public qwRegistry;
+  QWManager public qwManager;
+
+  error InvalidAddress(); // Error for invalid registry address
+
+  function run() public {
+    vm.startBroadcast();
+
+    // Sanity check deployment parameters.
+    DeploymentBaseParams memory baseParams = _readBaseEnvVariables();
+    ConfigParams memory configParams = _readEnvVariables();
+
+    qwManager = QWManager(baseParams.qwManager);
+
+    address registryAddr = qwManager.REGISTRY();
+    qwRegistry = QWRegistry(registryAddr);
+
+    // Deploy QwChild
+    qwAaveV2 = new QWAaveV2(baseParams.qwManager, configParams.aaveLendingPool);
```

Comment:
We should create offline registry at Contracts, could be json or .ts file which can be exported as well. Where we can add testnet and mainnet addresses for these deployments. Again reduces the overhead to add these contracts at env for every single person and for multiple packages as well.

Reply:
this makes sense, surely need .ts config file for list of deployed addresses
but again, don't get it why there is need to add addresses for every person
since [chain].env will be public

Reply:
chain.env.sample would be public, but that will be created by us. And the comment is contradictory to the env.sample setup.